### PR TITLE
mosquitto: add pt_BR translation

### PR DESCRIPTION
**Pull request by Guilospanck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_BR/common/mosquitto.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Um broker de MQTT.
+> Mais informaçōes: <https://mosquitto.org/>.
+
+- Iniciar mosquitto:
+
+`mosquitto`
+
+- Especificar um arquivo de configuração para usar:
+
+`mosquitto --config-file {{caminho/para/arquivo/configuracao.conf}}`
```
Comment (kbdharun) [inline pages.pt_BR/common/mosquitto.md:12]:
```suggestion
`mosquitto --config-file {{caminho/para/arquivo.conf}}`
```

File: pages.pt_BR/common/mosquitto.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Escutar em uma porta específica:
+
+`mosquitto --port {{8883}}`
+
+- Colocar como um processo rodando em background:
```
Comment (kbdharun) [inline pages.pt_BR/common/mosquitto.md:18]: I am not sure about this description. Is this correct?

> Note: I am not a native speaker, I checked this with Google Translate.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr @Guilospanck, I have some suggestions for this page.

File: pages.pt_BR/common/mosquitto.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# mosquitto
+
+> Um broker de MQTT.
+> Mais informaçōes: <https://mosquitto.org/>.
+
+- Iniciar mosquitto:
```
Comment (navarroaxel) [inline pages.pt_BR/common/mosquitto.md:6]: For Romance languages the present indicative fits better
```suggestion
- Inicia mosquitto:
```

File: pages.pt_BR/common/mosquitto.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`mosquitto`
+
+- Especificar um arquivo de configuração para usar:
```
Comment (navarroaxel) [inline pages.pt_BR/common/mosquitto.md:10]:
```suggestion
- Especifica um arquivo de configuração para usar:
```

File: pages.pt_BR/common/mosquitto.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`mosquitto --config-file {{caminho/para/arquivo/configuracao.conf}}`
+
+- Escutar em uma porta específica:
```
Comment (navarroaxel) [inline pages.pt_BR/common/mosquitto.md:14]:
```suggestion
- Escuta em uma porta específica:
```

File: pages.pt_BR/common/mosquitto.md (hunk 2 above)
Comment (navarroaxel) [reply pages.pt_BR/common/mosquitto.md:18]: Perhaps this better:
```suggestion
- Cria um processo rodando em background:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**